pallets/redeem/src/lib.rs

```diff
@@ -204,6 +204,10 @@ pub mod pallet {
 	#[pallet::storage]
 	pub(super) type LastIntervalIndex<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
 
+	#[pallet::storage]
+	pub(super) type CancelledRedeemAmount<T: Config> =
+		StorageMap<_, Blake2_128Concat, H256, (BalanceOf<T>, T::CurrencyId), OptionQuery>;
```

Comment:
Can we store this as `Amount<T>` or would that not work?

Reply:
No, that won't work
```rust
 the trait `WrapperTypeEncode` is not implemented for `currency::Amount<T>`

```